src/main/java/gregtech/api/nuclear/fission/components/ReactorComponent.java

```diff
@@ -0,0 +1,78 @@
+package gregtech.api.nuclear.fission.components;
+
+public class ReactorComponent {
+
+    private final double moderationFactor;
+    protected double maxTemperature;
+    private final double thermalConductivity;
+    private final double mass;
+
+    private int x;
+    private int y;
+
+    private final boolean isValid;
+
+    private int index = -1;
+
+    public ReactorComponent(double moderationFactor, double maxTemperature, double thermalConductivity, double mass,
```

Comment:
I mean, I never really understood what isValid was doing there anyways since Zalgo added it, so I think I kept it around in the case that it would be useful later. I've just removed it.
This definitely should not extend MTEMultiblockPart, given how a) this is meant to be easily transferable to modern versions and b) this is meant to handle two separate MTEs at once for the fuel rod and coolant components.